lib/iris/_representation/cube_printout.py

```diff
@@ -260,8 +260,11 @@ def add_scalar_row(name, value=""):
                 elif title in ("attributes:", "cell methods:", "mesh:"):
                     for title, value in zip(sect.names, sect.values):
                         add_scalar_row(title, value)
-                elif title == "scalar cell measures:":
-                    # These are just strings: nothing in the 'value' column.
+                elif title in (
+                    "scalar ancillary variables:",
+                    "scalar cell measures:",
+                ):
+                    # These are just strings: nothing in the 'va"scalar cell measures:"lue' column.
```

Comment:
```suggestion
                    # These are just strings: nothing in the "scalar cell measures:"l column.
```
Something weird happened in your edit !

Reply:
Not sure what happened here! I have now fixed this in the latest commit: https://github.com/SciTools/iris/pull/4945/commits/7673aa9c1d096fa59c64dab57c4d727ef170ed40

Reply:
I see my suggestion was confused, anyway.
Sorted now :+1: